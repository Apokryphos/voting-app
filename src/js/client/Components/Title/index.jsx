import React from 'react';
import PropTypes from 'prop-types';

function Title(props) {
  return <h1>{props.text}</h1>;
}

Title.propTypes = {
  text: PropTypes.string,
};

Title.defaultProps = {
  text: 'Untitled',
};

export default Title;
