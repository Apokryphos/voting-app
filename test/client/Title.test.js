import Title from 'Components/Title';
import React from 'react';
import ReactTestRenderer from 'react-test-renderer';

describe('Title rendering', () => {
  it('Renders OK', () => {
    const component = ReactTestRenderer.create(<Title />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
