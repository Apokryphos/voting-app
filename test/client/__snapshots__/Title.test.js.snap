// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title rendering Renders OK 1`] = `
<h1>
  Untitled
</h1>
`;
